StartMenuTest.py

```diff
@@ -0,0 +1,92 @@
+import sys
+from PySide6.QtWidgets import (
+    QApplication,
+    QWidget,
+    QVBoxLayout,
+    QHBoxLayout,
+    QPushButton,
+    QLabel,
+    QMessageBox,
+)
+from PySide6.QtCore import Qt
+from PySide6.QtGui import QPixmap, QPainter
+from MapScreen import MapScreen
+
+
+class StartMenu(QWidget):
+    def __init__(self):
+        super().__init__()  # Call the constructor of the parent class (QWidget)
+        self.background_image_path = "C:/Users/vuejohw/OneDrive - Dunwoody College of Technology/Documents/Data Structures/Start Menu Work/TitleScreen.jpg"  # Path to the background image
```

Comment:
Absolute paths won't work. We're going to need to include this in the `assets/` directory and import it dynamically.